### PR TITLE
Adds changes to radio group button.

### PR DESCRIPTION
**Pull request by mlauren**
## Original issue(s)
department-of-veterans-affairs/va-forms-system-core#420

## Definition of done
- [ ] Documentation has been updated, if applicable
- [ ] A link to the original github issue has been provided
- [ ] No sensitive information (i.e. PII/credentials/internal URLs etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: examples/burial-form/PreviousNames.jsx [hunk 1]
```diff
@@ -49,6 +50,7 @@ export default function PreviousNames(props) {
           }
         </div>
       </Page>
+      <DebuggerView />
```
Comment (bjmccotter7192) [inline examples/burial-form/PreviousNames.jsx:53]: Could you remove this?

File: examples/burial-form/utils.ts [hunk 2]
```diff
@@ -7,3 +7,19 @@ export const isBeforeDate = (
 ): () => string => {
   return () => isBefore(new Date(dateOne), new Date(dateTwo)) ? validationMessage : '';
 }
+
+export const stringToBoolean = (value: string): boolean => {
```
Comment (bjmccotter7192) [inline examples/burial-form/utils.ts:11]: In the branch this doesn't seem to be called anywhere, do we need this at all with this fix?

**Review by bjmccotter7192 — APPROVED** [review]
Looks good just had a couple clean up comments, but good work!

File: examples/burial-form/utils.ts (hunk 2 above)
Comment (harshil1793) [reply examples/burial-form/utils.ts:11]: Actually, we'll be using this method to transform the data while submit.

**Pull request merged**